packages/chain-adapters/src/utxo/UTXOChainAdapter.ts

```diff
@@ -93,7 +98,7 @@ export class ChainAdapter implements IChainAdapter<ChainTypes.Bitcoin> {
       {
         coin: this.coinName,
         addressNList: bip32ToAddressNList(path),
-        curve: 'secp256k1', // TODO(0xdef1cafe): from constant?
+        curve: this.curve,
```

Comment:
(yeah, hardcoding it is correct and it shouldn't be a constant either)

```suggestion
        curve: 'secp256k1',
```